pytorch_lightning/trainer/connectors/logger_connector/logger_connector.py

```diff
@@ -65,20 +64,21 @@ def should_update_logs(self) -> bool:
         return should_log_every_n_steps or self.trainer.should_stop
 
     def configure_logger(self, logger: Union[bool, LightningLoggerBase, Iterable[LightningLoggerBase]]) -> None:
-        if logger is True:
```

Comment:
Had to change the structure due to https://github.com/python/mypy/issues/11043

Reply:
It is cleaner this way :)